go.mod

```diff
@@ -10,11 +10,34 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/heketi/tests v0.0.0-20151005000721-f3775cbcefd6
 	github.com/lpabon/godbc v0.1.1
-	github.com/prometheus/client_golang v1.11.0
-	github.com/spf13/cobra v1.2.1
+	github.com/prometheus/client_golang v1.12.2
+	github.com/spf13/cobra v1.4.0
 	github.com/urfave/negroni v1.0.0
-	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
-	k8s.io/api v0.15.12
-	k8s.io/apimachinery v0.15.12
-	k8s.io/client-go v0.15.12
+	golang.org/x/crypto v0.0.0-20220525230936-793ad666bf5e
+	k8s.io/api v0.24.1
+	k8s.io/apimachinery v0.24.1
+	k8s.io/client-go v0.24.1
+)
+
+require (
+	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d // indirect
+	github.com/emicklei/go-restful/v3 v3.8.0 // indirect
+	github.com/form3tech-oss/jwt-go v3.2.5+incompatible // indirect
```

Comment:
Could you please make sure we don't include this one? I'm currently working on replace it with `golang-jwt/jwt` here: https://github.com/kubernetes/kubernetes/issues/104007

Reply:
@slaskawi rebased to master which picked that change.

Reply:
Awesome! Thank you!